source/grammar.tex

```diff
@@ -5,7 +5,7 @@
 \pnum
 \indextext{grammar}%
 \indextext{summary!syntax}%
-This summary of \Cpp\  syntax is intended to be an aid to comprehension.
+This summary of \Cpp\  grammar is intended to be an aid to comprehension.
```

Comment:
Also, this is actually in the wrong file, since this one is auto-generated from tools/grammarbase.tex.